wlm_server/task/measure.py

```diff
@@ -0,0 +1,45 @@
+"""Module for prioritized measurement queue."""
+
+import dataclasses
+import heapq
+from datetime import timedelta
+
+@dataclasses.dataclass
+class MeasureInfo:
+    """Measurement info.
+    
+    Fields:
+        channel: Target channel.
+        deadline: Measurement deadline.
+    """
+    channel: int
+    deadline: timedelta
+
+
+class MeasureQueue:
+    """Prioritized measurement queue based on deadlines."""
+
+    def __init__(self):
+        self._queue = []
```

Comment:
How about annotating the type here: `self._queue: list[tuple[datetime, MeasureInfo]] = []`?